go/vt/mysqlctl/schema.go

```diff
@@ -439,6 +439,11 @@ func (mysqld *Mysqld) ApplySchemaChange(ctx context.Context, dbName string, chan
 	}
 
 	sql := change.SQL
+
+	if change.SQLMode != "" {
+		sql = fmt.Sprintf("SET @@session.sql_mode='%s';\n%s", change.SQLMode, sql)
+	}
+
```

Comment:
:+1: pointing out to myself that this SQL is then written executed by `mysql` client, which terminates immediately after execution; hence, no need to worry about the session variables being changed.

Reply:
I'll add a comment on that too. Thanks!